src/main/java/sic/service/impl/CuentaCorrienteServiceImpl.java

```diff
@@ -489,8 +489,8 @@ private void cambiarFechaUltimoComprobante(CuentaCorriente cc, RenglonCuentaCorr
   @Override
   public byte[] getReporteCuentaCorrienteCliente(
       CuentaCorrienteCliente cuentaCorrienteCliente, String formato) {
-    ClassLoader classLoader = CuentaCorrienteServiceImpl.class.getClassLoader();
-    InputStream isFileReport =
+    var classLoader = CuentaCorrienteServiceImpl.class.getClassLoader();
```

Comment:
@k31kur0n0 remplazar por esta linea mas generica para no confundir el nombre de clase loader: var classLoader = this.getClass().getClassLoader();